android/src/main/java/com/swmansion/rnscreens/ScreenStackHeaderConfigViewManager.kt

```diff
@@ -201,4 +201,8 @@ class ScreenStackHeaderConfigViewManager : ViewGroupManager<ScreenStackHeaderCon
     override fun setDisableBackButtonMenu(view: ScreenStackHeaderConfig?, value: Boolean) {
         logNotAvailable("disableBackButtonMenu")
     }
+
+    override fun setBackButtonDisplayMode(view: ScreenStackHeaderConfig?, value: String?) {
+        logNotAvailable("backButtonDisplayMode")
+    }
```

Comment:
Does it work fine? Because I do not see `RNSScreenStackHeaderConfigManagerInterface<ScreenStackHeaderConfig>` being updated.

What we need to do here is run this project on Android Fabric (or just trigger the codegen manually from Android Studio) & copy generated `RNSScreenStackHeaderConfigManagerInterface<ScreenStackHeaderConfig>` interface from build folders `android/build/generate/source/codegen/java/com/facebook/react/viewmanagers/...` to `android/src/paper/java/com/facebook/react/viewmanagers/` so that the interfaces are up to date.

TODO: We need to automate this somehow.